includes/admin/downloads/metabox.php

```diff
@@ -887,7 +898,21 @@ function edd_render_file_row( $key, $args, $post_id, $index ) {
 		<?php do_action( 'edd_download_file_table_row', $post_id, $key, $args ); ?>
 
 	</div>
-<?php
+
+	<?php
+	if ( $custom_file_options ) {
+		?>
+
+		<div class="edd-custom-price-option-sections-wrap">
```

Comment:
I know this is using the class that it is to inherit the JS behavior without needing to modify the JS, but I would suggest adding an additional file specific class here so that it's a little less confusing when reading the code/markup.

Reply:
@robincornett I actually badly hurt my hand at the weekend, so typing (and coding) is a pain. Would you be able to make the required changes here please? Would help me a lot, as I am taking a break from coding until my hand recovers (also upgrading my mouse and keyboard to help).